sdk/eventgrid/Azure.Messaging.EventGrid/README.md

```diff
@@ -207,24 +213,36 @@ foreach (CloudEvent cloudEvent in cloudEvents)
             TestPayload testPayload = await cloudEvent.GetDataAsync<TestPayload>(myCustomSerializer);
             Console.WriteLine(testPayload.Name);
             break;
-        case "Microsoft.EventGrid.SubscriptionValidationEvent":
-            SubscriptionValidationEventData subscriptionValidated = cloudEvent.GetData<SubscriptionValidationEventData>();
-            Console.WriteLine(subscriptionValidated.ValidationCode);
+        case "Microsoft.Storage.BlobDeleted":
+            // Example for deserializing system events using GetData<T>
+            StorageBlobDeletedEventData blobDeleted = cloudEvent.GetData<StorageBlobDeletedEventData>();
+            Console.WriteLine(blobDeleted.BlobType);
             break;
     }
 }
 ```
 
 ## Troubleshooting
 
-#### Service Responses
+### Service Responses
 `SendEvents()` returns a HTTP response code from the service. A `RequestFailedException` is thrown as a service response for any unsuccessful requests. The exception contains information about what response code was returned from the service.
 
-#### Deserializing Event Data
+### Deserializing Event Data
 - An `InvalidCastException` will be thrown during `GetData<T>()` if the event data cannot be cast to the specified type.
 
 - An `InvalidOperationException` will be thrown during `GetData<T>()` if a custom serializer is passed into `GetData<T>()` with non-serialized event data (for example, if the event was created by the user and not created by parsing from JSON).
 
+### Setting up console logging
```

Comment:
We should link out to the documentation for this I think. Look at ServiceBus for example.

Reply:
For the exceptions, or the logging?

Reply:
The console logging. We link out to central documentation for this in other libraries.

Reply:
Oh the link is at the bottom (line 244)

Reply:
Here is the format in SB - https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/servicebus/Azure.Messaging.ServiceBus#other-exceptions